src/ClientProtos/user_defined_capture_info.proto

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```
Since you changed the other one.

Reply:
> Is it going to stay in `ClientProtos` even if it is for the new capture format?

Yes, the additional sections are going to use protos from this module.